ChartboostAdapter/src/main/java/com/chartboost/mediation/chartboostadapter/ChartboostAdapter.kt

```diff
@@ -145,10 +145,10 @@ class ChartboostAdapter : PartnerAdapter {
     override suspend fun setUp(
         context: Context,
         partnerConfiguration: PartnerConfiguration,
-    ): Result<Unit> {
+    ): Result<Unit> = withContext(IO) {
```

Comment:
the code is fine, but we might want to check the monetization sdk to see if this is a problem.